CMakeLists.txt

```diff
@@ -274,6 +281,12 @@ mark_as_advanced(UA_ENABLE_PUBSUB_BUFMALLOC)
 #RT and Transport PubSub settings
 option(UA_ENABLE_PUBSUB_ETH_UADP "Enable publish/subscribe UADP over Ethernet" OFF)
 
+if(UA_ENABLE_PUBSUB_ENCRYPTION)
```

Comment:
The TPM implementation should be a SecurityPolicy plugin that can coexist with software encryption.
**Do not make them incompatible**.